README.md

```diff
@@ -1,7 +1,7 @@
 cli-width
 =========
 
-Get stdout window width, with three fallbacks, `tty`, a custom environment variable and then a default.
+Get stdout window width, with four fallbacks, `tty`, a custom environment variable and then a default.
 
```

Comment:
Probably worth mentioning what the 4th fallback is. Maybe make this a numbered list instead?